rails/pluralization/gd.rb

```diff
@@ -0,0 +1,29 @@
+module RailsI18n
+  module Pluralization
+    module ScottishGaelic
+      def self.rule
+        lambda do |n|
+          return :other unless n.is_a?(Numeric)
+
+          floorn = n.floor()
```

Comment:
In Ruby we’d usually drop the parentheses here.

```suggestion
          floorn = n.floor
```